### PR TITLE
CR example

### PR DESCRIPTION
**Pull request by akanso**
targeting issue: #538

This example creates, lists, Gets, updates and deletes a custom resource.

### CODE REVIEW

**Comment by brendandburns** [comment]
Thanks for the PR. We need you to sign the CLA and then we can merge this.

**Comment by akanso** [comment]
> Thanks for the PR. We need you to sign the CLA and then we can merge this.

Thanks, I already did.

I realized my commits where under a different email, that's why there is one requirement still missing

File: examples/customResource/customResource.sln [hunk 1]
```diff
@@ -0,0 +1,18 @@
+﻿
```
Comment (tg123) [inline examples/customResource/customResource.sln:1]: please add this proj to root sln

**Review by tg123 — APPROVED** [review]

File: examples/customResource/customResource.sln (hunk 1 above)
Comment (akanso) [reply examples/customResource/customResource.sln:1]: done!

**Comment by brendandburns** [comment]
@akanso this is still not quite right, I don't think. One of the commits has 'Ubuntu' as the author, I suspect that is the problem.

**Comment by akanso** [comment]
closing in favor of #540

**Pull request closed without merging**